service.go

```diff
@@ -37,6 +39,15 @@ func (fis *fiServiceImpl) IDs() []string {
 	return UUIDs
 }
 
+func (fis *fiServiceImpl) apiUrls() []string {
+	var apiUrls = []string{}
+	for uuid := range fis.financialInstruments {
+		apiUrl := fis.baseUrl + uuid
```

Comment:
I expect constructing the url is the reponsibility of the handler and not the service itself... Wouldn't be better just to return the uuids here? What is your opinion?